pkg/version/version.go

```diff
@@ -31,8 +31,8 @@ import (
 var Version = "devel"
 
 const (
-	minK8sMajor = 1 // We need K8s 1.17 for endpoint slices
-	minK8sMinor = 17
+	minK8sMajor = 1  // We need K8s 1.17 for endpoint slices;
```

Comment:
```suggestion
	minK8sMajor = 1 // We need K8s 1.17 for endpoint slices;
```

Is this space extra or intentional?

Reply:
It aligns the comments.